packages/contracts/contracts/factories/DAOFactory.sol

```diff
@@ -92,49 +79,119 @@ contract DAOFactory {
         // TODO: shall we add minter as well ?
```

Comment:
Can this comment be removed?

Reply:
This we can discuss later on and do in another PR if really required.

Reply:
Ok